### ISSUE TITLE
crash connecting input of plugin 'eq6q'

### ISSUE THREAD

**Issue by Houston4444** [falkTX/Carla#239]
carla always crashes with 'eq10q' plugin connecting these jack ports: 
eq6q mono : input
eq6q stereo: input 2

It also does for eq6q and probably the other ones.
Terminal says:
Segmentation fault

**Comment by falkTX** [comment]
I have experienced this, but it's an issue with eq10q not carla.
I'll let it this bug report remain open though, we have the source so maybe I can try fixing it.

**Comment by kiselev-dv** [comment]
Hi, I've also fall into same troubles. 
I've tried some workarounds and what I've found:

I took EQ1M and have commented section for LV2 Atom messages reading and it fixes EQ1M plugin. I also have tried to substitute reading routines from examples, but it doesn't help. Only complete removing atom reading routine helps. So It doesn't seems to be an error in EQ1M code error.

At least I've got some extra information. After you connect audio line, atom control port also starting to receive messages. All of them are ignored because of their type (EQ waits for Atom Object message). EQ takes couple of messages with strange types (the are too big for mapped messages types) and then crashes. Sometimes it crashes on a first atom message. 

I can't  say for sure, but it seems like EQ crashes after receiving message with type that less than a zero. Maybe it reads atom feed at wrong moment or something else like that.
